jwst/outlier_detection/outlier_detection_step.py

```diff
@@ -115,30 +114,21 @@ def process(self, input_data):
             # Add logic here to select which version of OutlierDetection
             # needs to be used depending on the input data
             if self.input_container:
-                exptype = self.input_models[0].meta.exposure.type
+                single_model = self.input_models[0]
             else:
-                exptype = self.input_models.meta.exposure.type
+                single_model = self.input_models[0]
```

Comment:
This `if/else` seems to be doing the same thing in both branches.

Reply:
Good catch.  The else statement should not have `[0]` to match the behavior before this PR.  Fixed.